common/include/pcl/PCLHeader.h

```diff
@@ -1,14 +1,9 @@
 #pragma once
 
-#ifdef USE_ROS
-   #error USE_ROS setup requires PCL to compile against ROS message headers, which is now deprecated
-#endif 
-
-#include <string>
-#include <vector>
-#include <boost/shared_ptr.hpp>
-#include <pcl/pcl_macros.h>
-#include <ostream>
+#include <string>   // for string
+#include <ostream>  // for ostream
+
+#include <pcl/make_shared.h>  // for shared_ptr
```

Comment:
I thought we had converged on the order: pcl->3rd parties -> std

Reply:
Yeah. I am correcting the order in another PR. I didn't touch the order in this PR.